buffer/src/main/java/io/netty/buffer/api/bytebuffer/NioBuffer.java

```diff
@@ -538,7 +569,7 @@ public Buffer writeChar(char value) {
             woff += 2;
             return this;
         } catch (IndexOutOfBoundsException e) {
-            throw checkWriteState(e, woff);
+            throw checkWriteState(e, woff, 2);
```

Comment:
sad that there is no `Character.BYTES`. We could use `Character.SIZE / 2` but this doesn't sound better